sdk/test/utxo.test.ts

```diff
@@ -1,9 +1,19 @@
-import { describe, it, assert } from 'vitest';
+import { vi, describe, it, assert, Mock, expect } from 'vitest';
 import { AddressType, getAddressInfo, Network } from 'bitcoin-address-validation';
 import { Address, NETWORK, OutScript, Script, Transaction, p2sh, p2wpkh, selectUTXO } from '@scure/btc-signer';
 import { hex, base64 } from '@scure/base';
-import { createBitcoinPsbt, getInputFromUtxoAndTx, estimateTxFee } from '../src/wallet/utxo';
+import { createBitcoinPsbt, getInputFromUtxoAndTx, estimateTxFee, Input } from '../src/wallet/utxo';
 import { TransactionOutput } from '@scure/btc-signer/psbt';
+import { OrdinalsClient, OutPoint } from '../src/ordinal-api';
+
+vi.mock(import('@scure/btc-signer'), async (importOriginal) => {
+    const actual = await importOriginal();
+
+    return {
+        ...actual,
+        selectUTXO: vi.fn(actual.selectUTXO),
+    };
+});
```

Comment:
What is the purpose of this?

Reply:
it allows us to check what arguments it was called with.
- vi.fn - mock/spi function
- actual.selectUTXO - original function implementation

so when it will be called it will run the same code + we can spy on arguments/errors/return values